dataplanes/vpp/yaml/vpp-daemonset.yaml

```diff
@@ -14,8 +14,8 @@ spec:
         - name: vpp-daemon
           securityContext:
             privileged: true
-          image: networkservicemesh/vpp-daemon:latest
-          imagePullPolicy: Always
+          image: networkservicemesh/vppdataplane:latest
+          imagePullPolicy: IfNotPresent
```

Comment:
If you rebase on master, I believe you will find this changes has already been made :)  And some additional changes needed for the bind mounts to work correctly.